nfc/libs/NfcCoreLib/lib/Fri/phFriNfc_TopazDynamicMap.c

```diff
@@ -2008,7 +2010,7 @@ phFriNfc_Tpz_H_CheckCCBytesForWrite (
     ps_tpz_info = &(psNdefMap->TopazContainer);
 
     if ((PH_FRINFC_TOPAZ_CC_BYTE0 != ps_tpz_info->CCByteBuf[0]) ||
-        (TOPAZ_SPEC_VERSION != ps_tpz_info->CCByteBuf[1]) ||
+        (TOPAZ_SPEC_VERSION != (ps_tpz_info->CCByteBuf[1] & TOPAZ_MAJOR_VERSION_BIT_MASK)) ||
```

Comment:
Would it make sense to change this to `(TOPAZ_SPEC_VERSION & TOPAZ_MAJOR_VERSION_BIT_MASK) != (ps_tpz_info->CCByteBuf[1] & TOPAZ_MAJOR_VERSION_BIT_MASK)`?